adoc/headers/multipointer.h

```diff
@@ -21,10 +21,11 @@ enum class decorated : /* unspecified */ {
 } // namespace access
 
 // Shorthand aliases for address spaces
-constexpr inline access::address_space global_space = access::address_space::global_space;
-constexpr inline access::address_space local_space = access::address_space::local_space;
-constexpr inline access::address_space private_space = access::address_space::private_space;
-constexpr inline access::address_space generic_space = access::address_space::generic_space;
+using addrspace = access::address_space;
+constexpr inline addrspace addrspace_global = addrspace::global_space;
+constexpr inline addrspace addrspace_local = addrspace::local_space;
+constexpr inline addrspace addrspace_private = addrspace::private_space;
+constexpr inline addrspace addrspace_generic = addrspace::generic_space;
```

Comment:
Since this is no longer constrained by namespace hierarchy, why not following English like
`constexpr inline addrspace generic_addrspace = addrspace::generic_space;`
?

Reply:
@AerialMantis' suggestion was that we should always use `enum_value`, because that's the precedent elsewhere (e.g., `memory_order_relaxed`, `memory_scope_device`).

Reply:
But at the end I would prefer
```c++
constexpr inline addrspace global_as = addrspace::global_space;
constexpr inline addrspace local_as = addrspace::local_space;
constexpr inline addrspace private_as = addrspace::private_space;
constexpr inline addrspace generic_as = addrspace::generic_space;
```

Reply:
> @AerialMantis' suggestion was that we should always use `enum_value`, because that's the precedent elsewhere (e.g., `memory_order_relaxed`, `memory_scope_device`).

Does this impact this PR then?

Reply:
Yes, this is why we changed the name from `global_space` to `addrspace_global`.

If we were following established precedent very strictly we would have called the shorthand `addrspace_global_space`, but people didn't like the redundant `_space` at the end.  We had previously discussed renaming the `enum` values as well to avoid the `_space` suffix (see Khronos issue 654) but consensus was that renaming the address spaces was a step too far.